erpnext/crm/doctype/twitter_settings/twitter_settings.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import _
+from frappe.model.document import Document
+from frappe.utils.file_manager import get_file_path
+import os
+from frappe.utils import get_url_to_form, get_link_to_form
+import tweepy
```

Comment:
Add tweepy python package in requirements.txt, so that user don't have to manually install the package

Reply:
Added!